app/exceptions/Exceptions.scala

```diff
@@ -24,3 +24,5 @@ case class NoIncomeSourceFound(hash: String) extends RuntimeException(s"User has
 
 case class MultipleIncomeSourcesFound(hash: String, incomeSourceIds: List[String]) extends
   RuntimeException(s"User has multiple matching incomeSources. hash: <$hash>. incomeSourceIds: <$incomeSourceIds>")
+
+case class CouldNotCreateChargeItem(msg: String) extends RuntimeException(s"Could not create charge item: $msg")
```

Comment:
Please add suffix exception to the class name, => CouldNotCreateChargeItemException (or  ~RuntimeException ?)